frontend/src/v5/ui/controls/assigneesSelect/assigneesSelect.component.tsx

```diff
@@ -62,7 +62,7 @@ export const AssigneesSelect = ({
 	const teamspaceJobsAndUsers = UsersHooksSelectors.selectJobsAndUsersByIds();
 	const valueToJobOrUser = (val: string) => teamspaceJobsAndUsers[val];
 	const allJobsAndUsersToDisplay = [
-		...validValues.map(valueToJobOrUser),
+		...validValues.map((v) => valueToJobOrUser || ({ invalidItemName: v })),
```

Comment:
This `||` shouldn't be here: the "invalidItemName" should only be used if the value is invalid, so what's on the line below.
Also, you are no longer invoking the `valueToJobOrUser`